services-core/src/gas_price/eth_node.rs

```diff
@@ -13,6 +13,6 @@ impl GasPriceEstimating for Web3 {
             .gas_price()
             .await
             .map_err(From::from)
-            .map(num::u256_to_f64)
+            .map(|g| U256::to_f64_lossy(g))
```

Comment:
```suggestion
            .map(U256::to_f64_lossy)
```
Not sure if this works, but slight improvement if it does.

Reply:
Lets give it a try!